kornia/losses/ms_ssim_l1.py

```diff
@@ -124,7 +124,7 @@ def forward(self, yhat: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
         if not yhat.device == y.device:
             raise ValueError(f"input and target must be in the same device. Got: {yhat.device} and {y.device}")
 
-        self.g_masks = self.g_masks.to(yhat)
+        self.g_masks = cast(torch.Tensor, self.g_masks)
```

Comment:
tests failing when using `torch.device("cuda")`: 

`RuntimeError: Expected object of device type cuda but got device type cpu for argument #2 'weight' in call to _thnn_conv_depthwise2d_forward`

Maybe because `g_masks` is not being sent to GPU? Worked previously with `self.g_masks = self.g_masks.to(yhat)`.

Reply:
Okay, I was running the old file hahah!

Reply:
with register_buffer, you will need to `Module.cuda()`. probably we would keep the `self.g_masks.to(yhat)` to align with the rest part of our lib? @edgarriba